foundationdb-kubernetes-sidecar/files/GeoTrust_Global_CA.pem

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I put the file this directory so we only need to store it once (and Docker only allows to copy files from subdirectories or the same directory).